code/game/objects/items/storage/bags.dm

```diff
@@ -239,6 +239,10 @@
 		/obj/item/food/honeycomb,
 		/obj/item/seeds,
 		))
+
+/obj/item/storage/bag/plants/cyborg
+	name = "cyborg plant bag"
+
```

Comment:
I suppose this is also tgcode?

Reply:
This one is bubber actually. I should add comments probably right?

Reply:
You can put it into a modular file

Reply:
@Majkl-J got it